src/main/java/teammates/ui/template/StudentFeedbackResultsQuestionWithResponses.java

```diff
@@ -1,7 +1,14 @@
 package teammates.ui.template;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
+/**
+ * View Model for student/feedbackResults/questionWithResponses.tag
+ *
+ * <p>Manages student feedback results questions details, self-responses tables and others-responses tables separately
+ * for a specific question.
+ */
```

Comment:
As @wkurniawan07 mentioned in his comment some time ago, it may not be very meaningful to add JavaDoc for code in `ui.template` right now (though I personally think it's fine here :))

One question here though - why did you add `<p>` in the JavaDoc comment?

Reply:
I checked the coding standard for JavaDoc, and it informs that this is the correct way of adding a new paragraph if there's an empty line above. Otherwise it cannot pass the travis test. If it really does't look nice, I think it should be ok to just delete the entire JavaDoc.

Reply:
Leave it there for now then, unless @xpdavid feels otherwise :P